main.nf

```diff
@@ -44,205 +52,18 @@ Checks.aws_batch(workflow, params)
-     * SUBWORKFLOW: Call variants
-     */
-    VARIANTS_BCFTOOLS ( 
-        BAM_SORT_SAMTOOLS.out.bam,
-        ch_reference
-    )
-    ch_software_versions = ch_software_versions.mix(VARIANTS_BCFTOOLS.out.bcftools_version.first().ifEmpty(null))
-
-    /*
-     * MODULE: Make pseudogenome from VCF
-     */
-    VCF2PSEUDOGENOME (
-        VARIANTS_BCFTOOLS.out.filtered_vcf,
-        ch_reference
-    )
-    ch_software_versions = ch_software_versions.mix(VCF2PSEUDOGENOME.out.version.first().ifEmpty(null))
-
-    /*
-     * MODULE: make pseudogenome alignment
-     */
-    ALIGNPSEUDOGENOMES (
-        VCF2PSEUDOGENOME.out.pseudogenome.map { pseudogenome -> pseudogenome[1] }.collect(),
-        ch_reference
-    )
-    ch_software_versions = ch_software_versions.mix(ALIGNPSEUDOGENOMES.out.version.ifEmpty(null))
-    ALIGNPSEUDOGENOMES.out.aligned_pseudogenomes
-        .branch { 
-            aligned_pseudogenomes ->
-            ALIGNMENT_NUM_PASS: aligned_pseudogenomes[0].toInteger() >= 4
-            ALIGNMENT_NUM_FAIL: aligned_pseudogenomes[0].toInteger() < 4
-        }
-        .set { aligned_pseudogenomes_branch }
-    
-    // Don't proceeed further if two few genonmes
-    aligned_pseudogenomes_branch.ALIGNMENT_NUM_FAIL.view { "Insufficient (${it[0]}) genomes after filtering to continue. Check results/pseudogenomes/low_quality_pseudogenomes.tsv for details"}
-
-    aligned_pseudogenomes_branch.ALIGNMENT_NUM_PASS
-        .map{ it[1] }
-        .set { aligned_pseudogenomes }
-
-    SNPSITES(
-        aligned_pseudogenomes
-    )
-    ch_software_versions = ch_software_versions.mix(SNPSITES.out.version.ifEmpty(null))
-    
-    /*
-     * MODULE: remove recombination
-     */
-    if (params.remove_recombination){
-        GUBBINS (
-            aligned_pseudogenomes
-        )
-        ch_software_versions = ch_software_versions.mix(GUBBINS.out.version.ifEmpty(null))
-        /*
-        * SUBWORKFLOW: Create phylogenies
-        */
-        CREATE_PHYLOGENY (
-            GUBBINS.out.fasta,
-            SNPSITES.out.constant_sites_string
-        )
-    } else {
-        /*
-        * SUBWORKFLOW: Create phylogenies
-        */
-        CREATE_PHYLOGENY (
-            SNPSITES.out.fasta,
-            SNPSITES.out.constant_sites_string
-        )
-    }
-    ch_software_versions = ch_software_versions.mix(CREATE_PHYLOGENY.out.rapidnj_version.ifEmpty(null))
-    ch_software_versions = ch_software_versions.mix(CREATE_PHYLOGENY.out.fasttree_version.ifEmpty(null))
-    ch_software_versions = ch_software_versions.mix(CREATE_PHYLOGENY.out.iqtree_version.ifEmpty(null))
-    ch_software_versions = ch_software_versions.mix(CREATE_PHYLOGENY.out.raxmlng_version.ifEmpty(null))
-
-    /*
-     * MODULE: Pipeline reporting
-     */
-    GET_SOFTWARE_VERSIONS ( 
-        ch_software_versions.map { it }.collect()
-    )
-
+    include { BACTMAP } from './workflows/bactmap'
+    BACTMAP()
 }
 
 ////////////////////////////////////////////////////
```

Comment:
You can remove then this Completion email statment from `main.nf` as it is included inside the workflow.